site/content/faq/401-how-do-i-document-my-components.md

```diff
@@ -0,0 +1,32 @@
+---
+question: How do I document my .svelte files?
```

Comment:
For clarity, his should match the filename, either by renaming the file or modifying the title.